_posts/2020-07-10-removing-negative-words

```diff
@@ -0,0 +1,86 @@
+---
+layout: post
+title: "Removing Negative Words From our Codebase"
+date: 2020-07-10 11:00:00
+reviewed: 2020-07-10 11:00:00
+categories: ["inclusion", "culture"]
+author: "abizzinotto"
+---
+
+The discussion around the use of problematic words / terms in technology isn't a new one. This issue [can be traced back to, at least, 2003](https://edition.cnn.com/2003/TECH/ptech/11/26/master.term.reut/) when a Los Angeles county worker filed a complaint with the city's Office of Affirmative Action Compliance after seeing "master" and "slave" in computer equipment labels.
+
+This issue has resurfaced with recent events and, at OmbuLabs, we believe it is important to take it seriously. It is past time to remove negative words from our codebase. We have started taking the necessary steps to rename "master" branches to "main", as well as (and perhaps more importantly) remove any reference to the term "slave" and replace "whitelist / blacklist" with "allowlist / blocklist". Our team is actively working on that.
+
+But we understand that is not enough. This is not an action that effects change in its own. It's part of a larger commitment. In this article, I'll explain the context and motivation for this change and how it fits with our core values and log term vision as a company.
+
+<!--more-->
+
+## Context
+
+As far as the software community is concerned, efforts to move away from such terms have been underway for quite some time now. In 2014, [Drupal](https://www.drupal.org/) opted for the use of "primary / replica" instead of "master / slave" and, in the same year, [Django] (https://www.djangoproject.com/) opted to use "leader / follower". Then in 2018 the decision was made to remove the terminology from Python, same year that the IETF (Internet Engineering Task Force) published their [Terminology, Power and Oppressive Language](https://tools.ietf.org/id/draft-knodel-terminology-00.html#rfc.section.1.1) memo arguing for the terminology change and offering alternatives to be use instead of the offensive terms.
+
+Triggered by current events, the discussion has resurfaced, and recently several companies, big and small, and communities have been making the change (examples include OpenZFS and Go) or making a commitment to change, with GitHub's CEO claiming [in a Tweet](https://twitter.com/natfriedman/status/1271253144442253312) that his team has already been working on transitioning from "master" to "main" for default branch structure. The Android and Chrome teams have also been determined to replace "whitelist / blacklist" with "allowlist / blocklist".
+
+Not everyone in the community agrees with the change though. When Python moved away from the terminology in 2018, there was a heated discussion over it, with several members of the community questioning the change and it's real impact. Before that, the issue had already divided the Redis community.
```

Comment:
```suggestion
Not everyone in the community agrees with the change though. When Python moved away from the terminology in 2018, there was a heated discussion over it, with several members of the community questioning the change and its real impact. Before that, the issue had already divided the Redis community.
```

Reply:
Could you add a link to the comment about the Redis community? I remember there was a discussion about it but I don't remember what it was about.

Reply:
@etagwerker I'm pulling from several different sources including old forum discussions and a blog post from a maintainers, so I adding a single link there wouldn't be a good idea as that wouldn't represent the whole discussion. What I was planning to do was add a list of sources at the end of the article.